testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -566,3 +566,18 @@ def test_alignto_reorder_atomgroups():
     ref = u.atoms[[3, 2, 1, 0]]
     rmsd = align.alignto(mobile, ref, select='bynum 1-4')
     assert_allclose(rmsd, (0.0, 0.0))
+
+def test_alignto_same_universe():
+    # Alike Issue 1825
```

Comment:
Not sure what you mean by "Alike" here

Reply:
I mean that is a similar example proposed in issue #1825. The difference is that I used another lipid because the issue was raised 4 years ago. Should I erase "Alike"?

Reply:
Ah ok that makes more sense, yeah probably best to just say "Issue 1825", that's sufficient to get folks to look at the original issue.